server/gitrest/docker-compose.yml

```diff
@@ -6,6 +6,8 @@ services:
             - "3000:3000"
         volumes:
             - git:/home/node/documents
+        environment:
+            - external_storage_enabled
```

Comment:
Just to be clear, you need to override this in your repo by adding a new docker-compose file. Also this is a mapping with your config file. So if you have the following:
"externalStorage": {
        "endpoint": "http://externalStorage:3005"
        "enabled": false;
}

Then the environment should be "- externalStorage__enabled=true" (Notice the double underscore)